lib/Dialect/SPIRV/SPIRVOps.cpp

```diff
@@ -733,6 +733,34 @@ static LogicalResult verify(spirv::BitcastOp bitcastOp) {
   return success();
 }
 
+namespace {
+
+// Convert chained `spirv::BitcastOp` operations into one
```

Comment:
Doc comment - to be consistent with other patterns.

Reply:
s/Convert/Converts/